cluster-service/Makefile

```diff
@@ -8,6 +8,8 @@ CONSUMER_NAME ?= $(shell az aks list --query "[?tags.clusterType == 'mgmt-cluste
 KEYVAULT_NAME ?= $(shell az keyvault list --query "[?tags.aroHCPPurpose=='service'].name" -g  ${SVC_KV_RESOURCEGROUP} --output tsv)
 FPA_CERT_NAME ?= firstPartyCert
 AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID ?= "57e54810-3138-4f38-bd3b-29cb33f4c358"
+AZURE_ARM_HELPER_IDENTITY_CLIENT_ID ?= "2c6ca254-36bd-43c8-a7a8-fe880bc2c489"
```

Comment:
While not the end of the world, we should probably replace both of these hard-coded values with lookups instead similar to other places in our Makefiles, it seems the convention is:

```
SOME_CLIENT_ID ?= $(shell az identity show \
			-g ${RESOURCEGROUP} \
			-n SOME_CLIENT_NAME \
			--query clientId)
```

This will improve our security posture and make our automation resilient to turnover of these IDs in the event they are rotated for whatever reason.